File: Mechanics_of_Materials-Computer-Problems/Code-Mechanics-of-Mat-Semester03/ali/chapter-1/p1_c1.py
import math

tel = int(input("Total Elements= "))

for c in range(1, tel + 1):
    diameter = float(input(f'diameter E:{c} (m)= '))
    load = float(input(f'load applied E:{c} (N)= '))

    area = math.pi * (diameter / 2)**2
    average_stress = load / area

    print(f'\n El {c} : Force= {load:.2f} N,   Area= {area:.4e} m²,   Average Stress= {average_stress / 1e6:.4f} MPa')

File: Mechanics_of_Materials-Computer-Problems/Code-Mechanics-of-Mat-Semester03/kimya/chapter-1/p1_c1.m
clc; clear; close all;

tel = input("number of elements: ");

for i = 1:tel
    dia = input(['diameter E:'  num2str(i)  ' (m): ']);
    load = input(['load applied E:'  num2str(i)  ' (N): ']);

    area = pi * (dia / 2)^2;
    averageStress = (load / area) / 1e6;

    fprintf('\n     Element Number: %d ==> Force: %.2f N,   Area: %.4e m²,   Average Stress: %.4f MPa\n', i, load, area, averageStress);
end